crates/pathfinder/src/ethereum.rs

```diff
@@ -21,6 +22,22 @@ pub enum Chain {
     Goerli,
 }
 
+lazy_static::lazy_static! {
+    static ref MAINNET_CHAIN_ID: StarkHash = StarkHash::from(0x534e5f4d41494eu128);
+    static ref GOERLI_CHAIN_ID: StarkHash = StarkHash::from(0x534e5f474f45524c49u128);
+}
+
+impl Chain {
+    pub fn starknet_chain_id(&self) -> &'static StarkHash {
+        match self {
+            // SN_MAIN
+            Chain::Mainnet => &MAINNET_CHAIN_ID,
+            // SN_GOERLI
+            Chain::Goerli => &GOERLI_CHAIN_ID,
+        }
+    }
+}
+
```

Comment:
Feels weird having this live in `ethereum.rs`. Maybe time to move this out to a common place? `core.rs` maybe.

Reply:
Ok, I'll move `Chain` to `core.rs`.